docs/k8s-cluster/nvidia-network-operator.md

```diff
@@ -0,0 +1,233 @@
+     git clone https://github.com/NVIDIA/deepops.git
+     cd deepops/
+     ./scripts/setup.sh
+     vi config/inventory
+     ```
+    Configuring the Ansible inventory file by editing the "config/inventory" file,  and verify connectivity to all nodes.
+    > NOTE: Be warned that `/etc/hostname` and `/etc/hosts` on each host will be modified to the name(s) specified in the inventory file, so it is best to use the actual names of the hosts.
+
+    When modifying the inventory, if the hosts are not accessible from the management node by their hostname, supply an an `ansible_host` with its IP address. Example of the inventory file:
+
+    ```yml
+    # in config/inventory...
+    [all]
+    mgmt01     ansible_host=192.168.1.11
+    gpu01      ansible_host=192.168.2.11
+    gpu02      ansible_host=192.168.3.11
+    ...
+    [kube-master]
+    mgmt01
+    [kube-node]
+    gpu01
+    gpu02
+    ```
+  - Add or modify user(s) across cluster:
+    The ansible scripts assume a consistent user which has access to all nodes in the cluster.
+     > Note: If a user with the same username, uid, and password exists on each node, skip this step. It is critical for the user to exist with the same uid across all nodes.
+    
+     ```sh
+     # The default user is `nvidia` with password `deepops`
+     # Modify this user/password in config/group_vars/all.yaml as desired
+     vi config/group_vars/all.yml
+     ```
+
+     Run the users playbook to create/modify the user across all nodes.
+
+     ```sh
+     # NOTE: If SSH requires a password, add: `-k`
+     # NOTE: If sudo on remote machine requires a password, add: `-K`
+     # NOTE: If SSH user is different than current user, add: `-u <user>`
+     ansible-playbook -b playbooks/generic/users.yml
+     ```
+     Verify the configuration
+
+     ```sh
+     ansible all -m raw -a "hostname"
+     ```
+
+  - Deploying and verifying Kubernetes cluster
+    Install Kubernetes using Ansible and Kubespray
+
+    ```sh
+    # NOTE: If SSH requires a password, add: `-k`
+    # NOTE: If sudo on remote machine requires a password, add: `-K`
+    # NOTE: If SSH user is different than current user, add: `-u ubuntu`
+    ansible-playbook -l k8s-cluster playbooks/k8s-cluster.yml
+    ```
+    Please refer to [DeepOps Kubernetes Deployment Guidehere](https://github.com/NVIDIA/deepops/blob/master/docs/kubernetes-cluster.md) for more information.
+    
+    Verify that Kubernetes clustering is working with "kubectl get nodes" command:
+    ```sh
+    nvidia@mgmt01:~$ kubectl get nodes
+    NAME     STATUS   ROLES    AGE    VERSION
+    mgmt01   Ready    master   1d8h   v1.21.6
+    gpu01    Ready    <none>   1d8h   v1.21.6
+    gpu02    Ready    <none>   1d8h   v1.21.6
+    nvidia@mgmt01:~$
+    ```
+5. Deploy Nvidia Network Operator
+   Before runnng the playbook, please update "roles/nvidia-network-operator/vars/main.yml" file according to your hardware and network configuration, this is what we used in our value.yaml file:

```

Comment:
This is poor practice, configuration changes should be put into the config/group_vars directory, not directly into the role. We should update this if necessary and potentially update variable names to be particular for this role (i.e. network_operator_*)